diesel/src/main.rs

```diff
@@ -60,7 +60,7 @@ async fn add(
 
 /// This handler manually parse json object. Bytes object supports FromRequest trait (extractor)
 /// and could be loaded from request payload automatically
-async fn index_add(body: Bytes, pool: web::Data<Pool>) -> Result<HttpResponse, Error> {
```

Comment:
this won't work. `let mut body = BytesMut::new();` must be removed

Reply:
Fixed